llvmlite/tests/test_binding.py

```diff
@@ -173,6 +173,29 @@ def no_de_locale():
 """
 
 
+issue_632_elf = \
+    "7f454c4602010100000000000000000001003e00010000000000000000000000000000" \
+    "0000000000e0000000000000000000000040000000000040000500010048c1e2204809" \
+    "c24889d048c1c03d4831d048b90120000480001070480fafc800000000000000000000" \
+    "0000000000000000000000000000002f0000000400f1ff000000000000000000000000" \
+    "00000000070000001200020000000000000000001f00000000000000002e7465787400" \
+    "5f5f617279626f002e6e6f74652e474e552d737461636b002e737472746162002e7379" \
+    "6d746162003c737472696e673e00000000000000000000000000000000000000000000" \
+    "0000000000000000000000000000000000000000000000000000000000000000000000" \
+    "00000000000000001f0000000300000000000000000000000000000000000000a80000" \
+    "0000000000380000000000000000000000000000000100000000000000000000000000" \
+    "000001000000010000000600000000000000000000000000000040000000000000001f" \
+    "000000000000000000000000000000100000000000000000000000000000000f000000" \
+    "01000000000000000000000000000000000000005f0000000000000000000000000000" \
+    "0000000000000000000100000000000000000000000000000027000000020000000000" \
+    "0000000000000000000000000000600000000000000048000000000000000100000002" \
+    "00000008000000000000001800000000000000"
```

Comment:
Can a comment please be added saying that this hex sequence on x86_64 linux produces:
```
$ objdump -D 623.elf 

623.elf:     file format elf64-x86-64


Disassembly of section .text:

0000000000000000 <__brybo>:
        ...
  10:   00 48 b9                add    %cl,-0x47(%rax)
  13:   01 20                   add    %esp,(%rax)
  15:   00 04 80                add    %al,(%rax,%rax,4)
  18:   00 10                   add    %dl,(%rax)
  1a:   70 48                   jo     64 <__brybo+0x64>
  1c:   0f af c8                imul   %eax,%ecx
```
just as a reference for the contents, thanks.

Reply:
Sure!

Reply:
I'm finding that my output is different to yours:

```
$ objdump -D 623.elf 

623.elf:     file format elf64-x86-64


Disassembly of section .text:

0000000000000000 <__arybo>:
   0:	48 c1 e2 20          	shl    $0x20,%rdx
   4:	48 09 c2             	or     %rax,%rdx
   7:	48 89 d0             	mov    %rdx,%rax
   a:	48 c1 c0 3d          	rol    $0x3d,%rax
   e:	48 31 d0             	xor    %rdx,%rax
  11:	48 b9 01 20 00 04 80 	movabs $0x7010008004002001,%rcx
  18:	00 10 70 
  1b:	48 0f af c8          	imul   %rax,%rcx
```

How did you produce `623.elf`? I used

```python
raw = bytes.fromhex(issue_632_elf)

with open("623.elf", "wb") as f:
    f.write(raw)
```

Reply:
ah, I recall now, I manually modified the ELF when testing it, re-dumping the bytes gives the same as you!

Reply:
Reference output now added.

Reply:
Thanks!